### PR TITLE
5.27 tw epbds 13462

### PR DESCRIPTION
**Pull request by ekosolapova**

### CODE REVIEW

**Review by OlgaGeorgesEIS — CHANGES_REQUESTED** [review]
1) "array: The array into which elements are to be inserted." - inserted means any position, while this function adds to the end. I would replace inserted with 'added' in this sentence
2) .<br/>&nbsp;&nbsp;&nbsp;&nbsp;elements: One or... - the function has element in singular form (**add**(array,element)). replace with 'element'? (for both functions add and addElement
3) why did we change the description to 'Creates a copy of the given array' for add and addElement? **add**(array1, array2, ..., arrayN) and adAll do not create copy? I feel this 'creates a copy' only will add confusion. Is it technically more correct?
4) Creates a copy of the given array and inserts one or more specified elements of states - why 'states'?
5) The position in the array where the new elements should be inserted.  - replace should with 'must'
6) This is a zero-based index. - you mean count starts from '0'? maybe simplify the wording for better understanding to "This index starts at zero."?
7) "This is a varargs parameter, allowing for multiple elements to be specified." - I would replace 'specified' with 'listed' here. 
8)  plus  all elements of new - double space between 'plus' and 'all'?

**Comment by ekosolapova** [comment]
states  - US states, NY, AL etc

**Review by OlgaGeorgesEIS — APPROVED** [review]
